.travis.yml

```diff
@@ -78,6 +78,11 @@ jobs:
       before_script: wget https://phpbench.github.io/phpbench/phpbench.phar https://phpbench.github.io/phpbench/phpbench.phar.pubkey
       script: php phpbench.phar run -l dots --report=default
 
+    - stage: Coding standard
+      php: nightly
```

Comment:
I suggest using a stable PHP version instead. It is less likely to break PHPCS itself.

Reply:
That will be done on `v3` after we fix all CS violations